Controls/Cafe/Customer2.c

```diff
@@ -19,11 +23,25 @@ void Customer2_checkout(char** name, int* cost) {
  */
 void Task_Customer_2(void* p_arg) {
 	OS_ERR err;	// Make sure to check for errors and print the error code if not OS_ERR_NONE
+	CPU_TS ticks;
 
-	// Return the name and cost into these values from Customer2_checkout
+	OSMutexPost(&RegisterOccupied_Mutex, OS_OPT_POST_ALL, &err);
```

Comment:
What is this MutexPost for?

Reply:
oops, that was a line that i forgot to delete! i'll delete it and push the updated version